web/src/lib/components/asset-viewer/asset-viewer-nav-bar.svelte

```diff
@@ -186,7 +186,9 @@
                 on:click={() => dispatch('toggleArchive')}
                 text={asset.isArchived ? 'Unarchive' : 'Archive'}
               />
-              <MenuOption on:click={() => onMenuClick('asProfileImage')} text="As profile picture" />
+              {#if asset.type !== AssetTypeEnum.Video}
```

Comment:
Wouldn't it make more sense to check if it's an image?

Reply:
Great shout @mertalev I was too focused on the video aspect 👍